yang/frr-zebra.yang

```diff
@@ -1845,6 +1905,38 @@ module frr-zebra {
           "Link bandwidth informational parameter, in megabits.";
       }
       // TODO -- link-params for (experimental/partial TE use in IGP extensions)
+      container state {
+        config false;
```

Comment:
Should we have a container counters under state? This could capture
1. Interface up/down coutners
2. Traffic counters?

Reply:
Not sure,  counters are specific to `zebra` module?

Reply:
I'd think we'd want up/down and packet counts, at least (?)

Reply:
I agree but as Chirag said it should probably go into the base interface model

Reply:
@chiragshah6 you want to put them in base interface model as counters? I guess that is fine if that is what you meant.